deps/chakrashim/src/inspector/java-script-call-frame.h

```diff
@@ -57,18 +56,15 @@ class JavaScriptCallFrame {
   v8::Local<v8::Object> details() const;
 
   v8::MaybeLocal<v8::Value> evaluate(v8::Local<v8::Value> expression,
-                                     bool* isError);
+                                     bool returnByValue, bool* isError);
```

Comment:
This seems to be the node convention, the other method has them all on separate lines by coincidence in this case.  There's not enough room on the line to support any two of those parameters.